src/components/page/sectionBreak.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ImageLoader from '../site/imageLoader';
+import Block from './blocks/index';
+
+class SectionBreak extends React.PureComponent {
+  render() {
+    const { title, blocksGroups, shared: blockShared } = this.props;
+    const sectionBreakImagePath = '/images/section_break_celebration.gif';
+
+    return (
```

Comment:
basic page for section breaks. This builds on content blocks and is similar to the singleCol layout but simplified and can be customized easily.